### PR TITLE
Refactored Awol Script

### PR DESCRIPTION
**Pull request by mateuszstacel**
Fixed connection string, Refactored/CleanUp code.

### CODE REVIEW

File: AwolScript/AwolScript/Program.cs [hunk 1]
```diff
@@ -106,64 +111,59 @@ private static void SendEmailWithReportAwol(List<CC_Scheduling> data)
                         PayId = "BMG-";
                     }
 
-
-                    int totalAwol = db.CC_SchedulingSickLate.SqlQuery("SELECT *, [Shift-Start] as ShiftStart, [Shift-End] as ShiftEnd FROM CC_SchedulingSickLate where status = 'awol' and Interviewer = 'Isba Shaheen (237719)' and Date >= DATEADD(MONTH, -3, GETDATE())").ToList().Count;
+                    int totalAwol = db.CC_SchedulingSickLate.SqlQuery($"SELECT *, [Shift-Start] as ShiftStart, [Shift-End] as ShiftEnd FROM CC_SchedulingSickLate where status = 'awol' and Interviewer = '{item.Interviewer}' and Date >= DATEADD(MONTH, -3, GETDATE())").ToList().Count;
```
Comment (HamzahIqbal765) [inline AwolScript/AwolScript/Program.cs]: Is this secure from sql injection? double check with Mokbul

Comment (mokkymiah) [reply AwolScript/AwolScript/Program.cs]: @mateuszstacel Please use Dapper as @HamzahIqbal765 highlighted this is prone to SQL injection.

File: AwolScript/AwolScript/App.config [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
   </startup>
   <connectionStrings>
-    <add name="DbContext" connectionString="data source=192.100.50.14;initial catalog=CC_Data_WMS_LIVE;user id=sa;password=BMG,12;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient" />
+    <add name="DbContext" connectionString="data source=192.100.50.14;initial catalog=CC_Data_WMS_LIVE;user id=Mateuszs;password=W7suy38iezc6ku2x;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient" />
```
Comment (mokkymiah) [inline AwolScript/AwolScript/App.config]: Created a new SQL account or use the WMS account.

File: AwolScript/AwolScript/Program.cs [hunk 3]
```diff
@@ -106,64 +111,59 @@ private static void SendEmailWithReportAwol(List<CC_Scheduling> data)
...
-                    Manager = item.Team,
-                    EntryDate = DateTime.Now,
-                    IntSign = false,
-                    ManSign = false,
-                    Approve = false,
-                    Deny = false
-                };
-
-                newRangeToAdd.Add(newRecord);
+                 query += $@"INSERT INTO cc_schedulingsicklate
```
Comment (mokkymiah) [inline AwolScript/AwolScript/Program.cs]: use Dapper.

File: AwolScript/AwolScript/Program.cs (hunk 1 above)
Comment (mateuszstacel) [reply AwolScript/AwolScript/Program.cs]: done

File: AwolScript/AwolScript/App.config (hunk 2 above)
Comment (mateuszstacel) [reply AwolScript/AwolScript/App.config]: I have removed it at all don't needed for now entity framework all is using dapper.

File: AwolScript/AwolScript/Program.cs (hunk 3 above)
Comment (mateuszstacel) [reply AwolScript/AwolScript/Program.cs]: changed to dapper

**Pull request merged**